tests/mesolve/test_batching.py

```diff
@@ -56,3 +56,26 @@ def test_flat_batching(nL1, npsi0):
     broadcast_shape = jnp.broadcast_shapes(nH, nL1, npsi0)
     assert result.states.shape == (*broadcast_shape, ntsave, n, n)
     assert result.expects.shape == (*broadcast_shape, nEs, ntsave)
+
+
+def test_batching_boris():
```

Comment:
Will be renamed/removed in follow up PR

Reply:
I think it's valuable as a test, but maybe with smaller dimensions and batch dimensions to speed it up?